_config.yml

```diff
@@ -23,6 +23,12 @@ FLINK_STATEFUN_STABLE_SHORT: "3.1"
 FLINK_STATEFUN_GITHUB_URL: https://github.com/apache/flink-statefun
 FLINK_STATEFUN_GITHUB_REPO_NAME: flink-statefun
 
+FLINK_ML_VERSION_STABLE: 2.0.0
```

Comment:
If we search `state` in this file, there seems to be some content related to statefun which do not have similar Flink ML counterpart. Could you help add these? Thanks!

Reply:
Very thanks for the review~ I complete the `flink_ml_releases` and the corresponding part. The doc part could be updated after the flink-ml doc get online~